### ISSUE TITLE
For issues and pull requests, last activity time is displayed instead of creation time

### ISSUE THREAD

**Issue by TicClick** [octobox/octobox#3141]
Are you experiencing this issue in octobox.io or your own instance?
 - [x] octobox.io
 - [ ] My instance
 
**Problem**

recently I have returned to using octobox.io and soon found out that for all pull requests and issues I am getting notifications for, their last activity time (edits, comments, perhaps merges of `master` with their feature branches) is presented as creation time instead. Some examples:

- notification `26352502` (I assume the numeration is global): https://github.com/ppy/osu-web/pull/9280 is created on Sep 13th, but its creation time is currently "7 hours ago", which is the time when [the last comment](https://github.com/ppy/osu-web/pull/9280#issuecomment-1261760745) was made or edited ([notification screenshot](https://user-images.githubusercontent.com/4686101/193032790-8564dedc-45bd-446a-ab03-da3dbd5f63b7.png)).
- notification `26351948`: last edit time of https://github.com/ppy/osu-wiki/pull/8015 is displayed instead

 **Goal**

I'd like to see a clear indication that this is the update time. not much benefit in displaying creation time since it's already there in the first comment.

**Comment by TicClick** [comment]
another issue which I've just noticed is that all pull requests and issues' titles Are In Title Case for Some Reason, which also looks incorrect

(minor nit: I also believe it is spelled `pull request` with a space in between)

**Comment by andrew** [comment]
Yeah this css class shouldn't be there: https://github.com/octobox/octobox/blob/master/app/views/notifications/_thread_subject.html.erb#L1

And the last variable on this line needs humanizing: https://github.com/octobox/octobox/blob/master/app/views/notifications/_thread_subject.html.erb#L13

**Comment by andrew** [comment]
@TicClick Pushed a couple of fixes for those, deploying them to octobox.io now
